manifests/kots-config.yaml

```diff
@@ -838,6 +838,14 @@ spec:
           required: true
           type: text
 
+        ## OpenShift cluster deploy
+        - name: isOpenShiftCluster
+          title: OpenShift cluster deploy
+          help_text: |-
+            Enable if you want to deploy in OpenShift cluster.
+          type: bool
+          default: "0"
+
```

Comment:
Kots has a function to check if the cluster is an OpenShift one :) 
https://docs.replicated.com/reference/template-functions-static-context#distribution

Reply:
Done, I used the variable `platformDistribution`